invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/DocumentsDetails/components/DocumentMetadata/DocumentMetadata.js

```diff
@@ -24,7 +24,7 @@ export default class DocumentMetadata extends Component {
 
   toggleVisibility = () => this.setState({ visible: !this.state.visible });
 
-  _render_book_info() {
+  render_book_info() {
```

Comment:
```suggestion
  renderBookInfo() {
```
This file had a few other instances of snake case.